eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -253,7 +253,8 @@ jobs:
 
     - ${{ if eq(parameters.runTests, 'True') }}:
       - script: |
-          call $(sourcesPath)\build.cmd -ci -prepareMachine -test /bl:artifacts/log/Release/Test.binlog /p:TargetOS=${{ parameters.targetOS }} /p:TargetArchitecture=${{ parameters.targetArchitecture }} ${{ parameters.extraProperties }}
+          cd $(sourcesPath)
```

Comment:
nit: you can use `workingDirectory: <wd>` on the task instead.

Reply:
Thanks, updated.